Packs/GoogleChronicleBackstory/ReleaseNotes/2_0_16.md

```diff
@@ -0,0 +1,27 @@
+
+#### Incident Fields
+##### Changed the value of **unsearchable** from true to false, for the following incidents fields:
```

Comment:
Are you sure we should change it for all fields?

Reply:
i did in case what happened with this one might happen with the others